### DISCUSSION TITLE
i have nat ip .static ip required ?

### DISCUSSION THREAD

**Discussion by erdempome** [Chia-Network/chia-blockchain#8712] [Support]
![kkkk](https://user-images.githubusercontent.com/89322274/135920625-5e17f103-c9ca-460a-a922-986765ba4b76.png)

**Comment by marssystems** [comment]
Ask your ISP to provide you with a static IP - there may be an extra charge for this.

**Reply by erdempome** [reply]
but now she is farming

**Reply by marssystems** [reply]
That is good!

**Reply by erdempome** [reply]
Are there any delays? 0 blocks

**Comment by BrandtH22** [comment]
@erdempome ,based on the previous messages it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
